src/components/FormElementLabelContainer.tsx

```diff
@@ -9,16 +9,19 @@ function FormElementLabelContainer({
   id,
   required,
   children,
+  childrenFirst,
 }: {
   className: string
   element: FormTypes.FormElementBase
   id: string
   required: boolean
   children: React.ReactNode
+  childrenFirst?: boolean
 }) {
   return (
     <div className={clsx('ob-form__element', className)}>
       <div className="label ob-label__container">
+        {childrenFirst && children}
```

Comment:
Is this going to put the validation message(s) between the label and the switch?

Prob better to not use `<FormElementLabelContainer />` component at all. As i expect we dont really want the `*` on the label if its required.